shared_models/shared_models.py

```diff
@@ -100,13 +103,54 @@ class Endorsement(BaseModel):
     state: str
 
 
+class ServiceDecorator(TypedDict):
+    endpoint: str
+    recipient_keys: List[str]
+    routing_keys: Optional[List[str]]
+
+
+class Oob(BaseModel):
+    role: Optional[str] = str
+    invi_msg_id: Optional[str] = None
+    state: Optional[str] = str
+    trace: Optional[str] = None
+    our_recipient_key: Optional[str] = None
+    oob_id: Optional[str] = None
+    invitation: Union[InvitationMessage, Mapping[str, Any]] = None
+    created_at: Optional[str] = None
+    connection_id: Optional[str] = None
+    updated_at: Optional[str] = None
+    attach_thread_id: Optional[str] = None
+    multi_use: Optional[bool] = False
+    our_recipient_key: Optional[str] = None
+    their_service: Optional[Union[ServiceDecorator, Any]] = None
+    our_service: Optional[Union[ServiceDecorator, Any]] = None
+
+
+class OobRecord(BaseModel):
```

Comment:
Why do we need to type OobRecord? Is this not exported from ACA-Py? (due to the bug?)

Reply:
Couldnt import that from version 0.7.4